apps/api/src/app/integrations/usecases/check-integration/check-integration-email.usecase.ts

```diff
@@ -10,10 +10,10 @@ export class CheckIntegrationEMail {
     const mailHandler = mailFactory.getHandler(
       {
         channel: command.channel,
-        credentials: command.credentials,
+        credentials: command.credentials ?? {},
         providerId: command.providerId,
       },
-      command.credentials.from
+      command.credentials?.from
```

Comment:
What about rather than do safechecks and fallbacks we don't throw an error when trying to get this method executed while missing the credentials? Might be something to implement in a different task though. 👍🏻